include/ebpf_util.h

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
eBPF-for-windows coding conventions is to avoid abbreviations.  Can we rename this file to ebpf_utilities.h or something similar?